java-network/src/main/scala/com/linkedin/norbert/javacompat/network/BaseNettyNetworkClient.scala

```diff
@@ -144,11 +146,8 @@ class NettyPartitionedNetworkClient[PartitionedId](config: NetworkClientConfig,
                            (responseIterator: ResponseIterator[ResponseMsg]) => scatterGather.gatherResponses(responseIterator))(serializer, serializer)
   }
 
- // def sendRequest[RequestMsg, ResponseMsg](requestSpec: com.linkedin.norbert.network.RequestSpecification[RequestMsg],nodeSpec: com.linkedin.norbert.network.client.NodeSpec,retrySpec: com.linkedin.norbert.RetrySpecifications[ResponseMsg], serializer:Serializer[RequestMsg, ResponseMsg]) =
-  //  underlying.sendRequest(requestSpec, nodeSpec, retrySpec)(serializer, serializer)
-  //(requestSpec: PartitionedRequestSpecification[RequestMsg, PartitionedId], nodeSpec: PartitionedNodeSpec[PartitionedId], retrySpec: PartitionedRetrySpecifications[ResponseMsg]
-  //com.linkedin.norbert.network.PartitionedRequestSpecification[RequestMsg(in method sendRequest),PartitionedId],nodeSpec: com.linkedin.norbert.network.client.PartitionedNodeSpec[PartitionedId],retrySpec: com.linkedin.norbert.PartitionedRetrySpecifications[ResponseMsg
-  def sendRequest[RequestMsg, ResponseMsg](requestSpec: com.linkedin.norbert.network.PartitionedRequestSpecification[RequestMsg, PartitionedId], nodeSpec: com.linkedin.norbert.network.client.PartitionedNodeSpecification[PartitionedId], retrySpec:com.linkedin.norbert.PartitionedRetrySpecification[ResponseMsg], serializer:Serializer[RequestMsg, ResponseMsg]) =
+
+  def sendRequest[RequestMsg, ResponseMsg](requestSpec: PartitionedRequestSpecification[RequestMsg, PartitionedId], nodeSpec: PartitionedNodeSpecification[PartitionedId], retrySpec:PartitionedRetrySpecification[ResponseMsg, Unit], serializer:Serializer[RequestMsg, ResponseMsg]) =
```

Comment:
(Already fixed in code, just commenting for posterity that this should be nodeSpecification, not nodeSpec.)